### PR TITLE
Update whitelisting guide

### PR DESCRIPTION
**Pull request by biancabuzea200**
- adds SelfKisser addresses on each chain for better visibility and convenience

### CODE REVIEW

**Comment by max-wickham** [comment]
Don;t know if they should be added or not, but I think the cast send commands wont run without --private-key or --keystore set

**Comment by pmerkleplant** [comment]
> I think the cast send commands wont run without --private-key or --keystore set

@max-wickham If no wallet argument provided `cast` will use default env variables (eg `ETH_KEYSTORE`).

Personally I think its fine to not "dictate" wallet management. If a user didn't set their default, they'll get an error and can decide themselves how they wanna manage wallets (private key, keystore, ledger...).

File: docs/Developers/Guides/whitelistAddress.md [hunk 1]
```diff
@@ -91,14 +131,14 @@ Make sure you are connected to the correct network corresponding to the contract
...
+
 <div style={{textAlign: 'center'}}>
 <img
     src="/img/guides/tolled.png"
     alt="Tolled" 
 />
 </div>
-Once you verify that you are whitelisted, you are ready to read from smart contracts.
\ No newline at end of file
+Once you verify that you are whitelisted, you are ready to read from smart contracts.
```
Comment (pmerkleplant) [inline docs/Developers/Guides/whitelistAddress.md:144]: > you are ready to read from smart contracts 

"ready to read from the oracle smart contract" (?)

Comment (pmerkleplant) [reply docs/Developers/Guides/whitelistAddress.md:144]: How to check whether whitelisted?

`$ cast call $oracle "tolled(address)(bool)" $my_address --rpc-url $rpc`

**Review by pmerkleplant — CHANGES_REQUESTED** [review]

File: docs/Developers/Guides/whitelistAddress.md (hunk 1 above)
Comment (biancabuzea200) [reply docs/Developers/Guides/whitelistAddress.md:144]: @pmerkleplant, the second option shows how to verify your whitelisted status using Etherscan
The following command:
`$ cast call $oracle "tolled(address)(bool)" $my_address --rpc-url $rpc` is presented part of Option 1: Using the CLI. I don't think we need to add it again here.

**Review by pmerkleplant — APPROVED** [review]

**Pull request merged**